### PR TITLE
Needs some description of how to access apps, channels, and favorites

### PR DESCRIPTION
**Pull request by scstraus**
If you can make this process somehow simpler, I'm all for it, but it's way too hard to figure out without some hint, so I do firmly believe that some help needs to be here to get you started. The fact that it doesn't mention that this stuff is exposed as media is a major oversight IMHO.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @scstraus, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20395).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/emulated_roku.markdown [hunk 1]
```diff
@@ -123,6 +123,26 @@ The following is an example implementation of an automation:
       entity_id: media_player.amplifier
 ```
 
+## Selecting apps, channels, or favorites
+
+Apps, channels, and favorites are exposed as media content to the integration. You can see which ones are available by clicking the media button in the media player more info card for your tv. To capture the content_id for the one you want to use in an automation or script, turn your logging on to debug level and tail the log while choosing the media content in the media player more info card. You will find a log message that looks like this when you choose the media:
```
Comment (frenck) [inline source/_integrations/emulated_roku.markdown]:
```suggestion
Apps, channels, and favorites are exposed as media content to the integration. You can see which ones are available by clicking the media button in the media player more info card for your tv. To capture the `content_id` for the one you want to use in an automation or script, turn your logging on to debug level and tail the log while choosing the media content in the media player more info card. You will find a log message that looks like this when you choose the media:
```

File: source/_integrations/emulated_roku.markdown [hunk 2]
```diff
@@ -123,6 +123,26 @@ The following is an example implementation of an automation:
       entity_id: media_player.amplifier
 ```
 
+## Selecting apps, channels, or favorites
+
+Apps, channels, and favorites are exposed as media content to the integration. You can see which ones are available by clicking the media button in the media player more info card for your tv. To capture the `content_id` for the one you want to use in an automation or script, turn your logging on to debug level and tail the log while choosing the media content in the media player more info card. You will find a log message that looks like this when you choose the media:
```
Comment (frenck) [inline source/_integrations/emulated_roku.markdown]:
```suggestion
Apps, channels, and favorites are exposed as media content to the integration. You can see which ones are available by clicking the media button in the media player more info card for your TV. To capture the `content_id` for the one you want to use in an automation or script, turn your logging on to debug level and tail the log while choosing the media content in the media player more info card. You will find a log message that looks like this when you choose the media:
```

File: source/_integrations/emulated_roku.markdown [hunk 3]
```diff
@@ -123,6 +123,26 @@ The following is an example implementation of an automation:
       entity_id: media_player.amplifier
 ```
 
+## Selecting apps, channels, or favorites
+
+Apps, channels, and favorites are exposed as media content to the integration. You can see which ones are available by clicking the media button in the media player more info card for your TV. To capture the `content_id` for the one you want to use in an automation or script, turn your logging on to debug level and tail the log while choosing the media content in the media player more info card. You will find a log message that looks like this when you choose the media:
+
+```
```
Comment (frenck) [inline source/_integrations/emulated_roku.markdown]:
````suggestion
```txt
````

**Review by frenck — APPROVED** [review]
Thanks, @scstraus 👍

**Pull request merged**